<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QWidget" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="clear_PB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光行书_CNKI</family>
    </font>
   </property>
   <property name="text">
    <string>CLC</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>258</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="IP_lineEdit">
        <property name="font">
         <font>
          <family>华光行书_CNKI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>192.168.0.119</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connect_PB">
        <property name="font">
         <font>
          <family>华光行书_CNKI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="Port_lineEdit">
        <property name="font">
         <font>
          <family>华光行书_CNKI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>12927</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnect_PB">
        <property name="font">
         <font>
          <family>华光行书_CNKI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="Messages_QTE">
      <property name="font">
       <font>
        <family>华光行书_CNKI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
